lib/classes/Swift/Transport/AbstractSmtpTransport.php

```diff
@@ -395,27 +413,6 @@ protected function streamMessage(Swift_Mime_SimpleMessage $message)
         $this->executeCommand("\r\n.\r\n", [250]);
     }
 
-    /** Determine the best-use reverse path for this message */
-    protected function getReversePath(Swift_Mime_SimpleMessage $message)
```

Comment:
As this one is protected, it should be deprecated and just modified to use the new method.

Reply:
The new method is also protected, so it cannot be called from here. Instead I just restored the existing code and marked it as deprecated.